packages/@adobe/spectrum-css-temp/components/menu/index.css

```diff
@@ -176,7 +176,10 @@ governing permissions and limitations under the License.
 }
 .spectrum-Menu-icon {
   grid-area: icon;
-  align-self: center;
+  /* This is a bandaid fix to align the icons with the top of multiline text items. */
+  /* It will need to be revisited when we actually pull in typography. */
+  vertical-align: top;
+  padding-block-start: 2px;
```

Comment:
Neither this 2px or other padding after this have spectrum variable values to use?

Reply:
For the 2px here, it doesn't for now since it is simply a bandaid fix. When we bring in typography and get access to things like font sizes/font ascender/font descender/etc then a more accurate calculation can be made w/ variables.